infra/run-sollya.rkt

```diff
@@ -63,7 +63,7 @@
                                    ;(truc . Truncate)                               ; no truncate
                                    (round . "round(~a)")
                                    (assert . "if (~a) then (~a) else nan")
-                                   (if . "(if (~a) then (~a) else (~a))")
+                                   (if . "if (~a) then (~a) else (~a)")
```

Comment:
Are you sure this doesn't break something with order of operations? I thought we had to put these parens in as part of a bug...

Reply:
Yes, Sollya actually crashes when using parens here: "(if (~a) then (~a) else (~a))".
By Sollya's syntax `if` statement can not be used like a _subexpression_.
Hopefully, we use `if` statement only once and it works fine